src/EntityFrameworkCore.Projectables.Abstractions/ProjectableAttribute.cs

```diff
@@ -23,5 +33,10 @@ public sealed class ProjectableAttribute : Attribute
         /// or null to get it from the current member.
         /// </summary>
         public string? UseMemberBody { get; set; }
+
+        /// <summary>
+        /// Parameters values for UseMemberBody.
+        /// </summary>
+        public object[]? MemberBodyParameterValues { get; set; }
```

Comment:
I would prefer the name `UseMemberBodyArguments`

Reply:
Comment accepted